.storybook/preview.ts

```diff
@@ -36,13 +36,18 @@ const parameters = {
   },
   options: {
     storySort: {
-      method: '',
+      method: 'alphabetical',
       order: [
         'Overview',
         'Developer Guide',
-        'Sample Pages (WIP)',
+        'Components',
         'Contexts',
-        'Packages',
+        'Hooks',
+        'LeafyGreen Provider',
+        'Demo',
+        'Form',
+        'Lib Utilities',
+        'Sample Pages (WIP)',
```

Comment:
It feels like these should be repo specific - `Demo` is really just from LG Chat, right?

Reply:
Yeah, the `preview.ts` parameters is statically analyzed, so it needs to be repo-specific